pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -181,7 +181,7 @@ def cache_training_step_metrics(self, opt_closure_result):
         self.logged_metrics.update(logged_metrics_tmp)
         self.cached_results.legacy_batch_log_metrics.update(logged_metrics_tmp)
 
-    def log_metrics(self, metrics, grad_norm_dic, step=None):
+    def log_metrics(self, metrics, grad_norm_dic, step=None, is_train_step=False):
```

Comment:
can we add docs what the`is_train_step` means?
```suggestion
    def log_metrics(self, metrics, grad_norm_dic, step=None, log_on_step=False):
```